.github/workflows/build-test-package.yml

```diff
@@ -2,141 +2,40 @@ name: Build, test, package
 
 on: [push,pull_request]
 
+env:
+  itk-git-tag: "v5.3rc04"
+  itk-wheel-tag: "v5.3rc04.post2" # Same ITK C++ reference commit, different tag name
+
 jobs:
   build-test-cxx:
     runs-on: ${{ matrix.os }}
     strategy:
       max-parallel: 3
       matrix:
-        os: [ubuntu-18.04, windows-2019, macos-10.15]
+        os: [ubuntu-20.04, windows-2019, macos-11]
         include:
-          - os: ubuntu-18.04
-            c-compiler: "gcc"
-            cxx-compiler: "g++"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
+          - os: ubuntu-20.04
             cmake-build-type: "MinSizeRel"
           - os: windows-2019
-            c-compiler: "cl.exe"
-            cxx-compiler: "cl.exe"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
             cmake-build-type: "Release"
-          - os: macos-10.15
-            c-compiler: "clang"
-            cxx-compiler: "clang++"
-            itk-git-tag: "d6acfd26bfcdec606d605beb1301bddfb17c05a6"
+          - os: macos-11
             cmake-build-type: "MinSizeRel"
 
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
 
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v1
+    - name: Build, test cxx
+      uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction@main
```

Comment:
While this is fine to use for now please note that we may move towards a reusable workflow instead of a custom action once I can get back to it. [https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/pull/3#discussion_r961711970](https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/pull/3#discussion_r961711970)